.github/workflows/live-links.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
             https://ubuntu.com/security
             http://www.brendangregg.com
             http://www.adlinktech.com/en/index
+            */flags.png
```

Comment:
Will this catch the links as they end with `/21572c97-flags.png` & `/d6f84371-flags%402x.png` respectively